src/taskpane/server.js

```diff
@@ -2622,10 +2889,9 @@ function updateSheetWithExportResults(entriesLog, commentsLog, associationsLog,
           }
         }
       }
-
       if (IS_GOOGLE) {
         rowBgColors.push(bgColor);
-        if (associationNote != null) {
+        if (associationNote.length > 0) {

```

Comment:
If there is any concern with associationNote being null,

 `if (associationNote?.length){}` 

is really nice syntax - if length is 0 it is falsy and if it is null, it evaluates to null which is also falsy. Makes the conditional safe for null and empty arrays, whereas now a null array would throw an error.

Reply:
Thanks for the suggestion! I'll check if Google's console will recognize this (most of the cool functions are not, unfortunately)